cli/admin/src/main/java/io/pravega/cli/admin/utils/SerializerUtils.java

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.cli.admin.utils;
+
+import com.google.common.collect.ImmutableMap;
+import io.pravega.segmentstore.contracts.SegmentProperties;
+import io.pravega.segmentstore.storage.metadata.ChunkMetadata;
+import io.pravega.segmentstore.storage.metadata.ReadIndexBlockMetadata;
+import io.pravega.segmentstore.storage.metadata.SegmentMetadata;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+
+/**
+ * Utility class for serialization purposes.
+ */
+public class SerializerUtils {
```

Comment:
I suggest getting rid of this class and
1. Creating an `AbstractSerializer` class with common methods.
2. Putting all these maps in the specific serializer classes that use them. Let's not have two related things living in different places.
3. Write unit tests for these methods. Testing them indirectly is not comprehensive enough.

Also please use constants instead of these strings. I see you use them both here and in the serializers, so if you change it in one place you won't have to remember to change them in the other one as well.

Reply:
Created `AbstractSerializer`. And added unit tests for these methods. Also added constants as suggested